src/components/SuccessContent.tsx

```diff
@@ -1,16 +1,38 @@
 import propTypes from 'prop-types';
+import { useState } from 'react';
 import { Trans } from 'react-i18next';
 
+import { MUTATION_KEYS } from '@graasp/query-client';
 import { AUTH, namespaces } from '@graasp/translations';
+import { Button } from '@graasp/ui';
 
 import MailOutlineIcon from '@mui/icons-material/MailOutline';
-import { Container, Typography } from '@mui/material';
+import { Container, Stack, Typography } from '@mui/material';
 
 import { useAuthTranslation } from '../config/i18n';
+import { useMutation } from '../config/queryClient';
 import { SUCCESS_CONTENT_ID } from '../config/selectors';
+import { BACK_BUTTON_ID, RESEND_EMAIL_BUTTON_ID } from '../config/selectors';
 
-const SuccessContent = ({ title, email }) => {
+const SuccessContent = ({ title, email, handleBackButtonClick = null }) => {
   const { t } = useAuthTranslation();
+  const [clicked, setClicked] = useState(false);
```

Comment:
For boolean, it's better to have `is` at the beginning of the variable, so `isClicked` and `setIsClicked`

Reply:
From your use of `clicked` maybe it's better to call it `isEmailSent`